File: TrueVote.Services/Database/Stranka.cs
using System;
using System.Collections.Generic;

namespace TrueVote.Services.Database;

public partial class Stranka
{
    public int Id { get; set; }

    public string Naziv { get; set; } = null!;

    public string? Opis { get; set; }

    public DateTime? DatumOsnivanja { get; set; }

    public int? BrojClanova { get; set; }

    public string? Sjediste { get; set; }

    public string? WebUrl { get; set; }

    public byte[]? Logo { get; set; }

    public virtual ICollection<Kandidat> Kandidats { get; set; } = new List<Kandidat>();
}

File: UI/truevote_mobile/lib/models/stranka.dart
import 'package:json_annotation/json_annotation.dart';

part 'stranka.g.dart';

@JsonSerializable()
class Stranka {
  int id;
  String? naziv;
  String? opis;
  DateTime? datumOsnivanja;
  String? sjediste;
  String? webUrl;
  String? logo;

  Stranka({
    required this.id,
    this.naziv,
    this.opis,
    this.datumOsnivanja,
    this.sjediste,
    this.webUrl,
    this.logo,
  });

  factory Stranka.fromJson(Map<String, dynamic> json) =>
      _$StrankaFromJson(json);

  Map<String, dynamic> toJson() => _$StrankaToJson(this);
}
